roles/speedtest-tracker/tasks/main.yml

```diff
@@ -6,9 +6,15 @@
   with_items:
     - "{{ speedtest_data_directory }}/config"
 
-- name: Speedtest-Tracker Docker Container
+- name: Remove old Speedtest-Tracker Docker Container
   docker_container:
     name: speedtest_tracker
+    state: absent
+  when: speedtest_tracker_enabled == true
```

Comment:
Aside from breaking the build, this line isn't required as it's covered by the `speedtest_tracker_enabled` applied at the role level.